class TestException(Exception):
    def __init__(self):
        self.message = 'Test Error'